libtextsecure/outgoing_message.js

```diff
@@ -13,14 +13,14 @@ const NUM_SEND_CONNECTIONS = 3;
 
 const getTTLForType = type => {
```

Comment:
this file is removed on clearnet. Be sure to remove those change so you do not have conflicts creating it back in.